plugins/lsp/language-servers/default.nix

```diff
@@ -234,6 +234,12 @@ with lib; let
       name = "gopls";
       description = "gopls for Go";
     }
+    {
+      name = "golangci-lint";
+      description = "golangci-lint-langserver for Go";
+      serverName = "golangci_lint_ls";
+      package = pkgs.golangci-lint-langserver;
```

Comment:
I'd suggest going with the nvim-lspconfig name

Reply:
Done in abb6e17f6832fa4ea53f7ec67c5eb25e392154d3